### ISSUE TITLE
Gradient Issue

### ISSUE THREAD

**Issue by wolfball** [VincentHancoder/REPARO#5]
I am running the command:

```bash
python experiments/furniture/optim_reparo.py --config config/dreamgaussian/furniture_rgbxy_wo_proxy.json
```

But the gradients are not being backpropagated to the optimizable parameters.

I have tried modifying "DcDt" to True in Line 271, but it does not work.

**Comment by VincentHancoder** [comment]
Hi, I wonder which optimizable parameters are the gradients not being backpropagated? Are both the rotation and translation parameters not being optimized?

**Comment by wolfball** [comment]
Yes, neither the rotation nor the translation parameters are being optimized.

> Hi, I wonder which optimizable parameters are the gradients not being backpropagated? Are both the rotation and translation parameters not being optimized?

**Comment by VincentHancoder** [comment]
OK, could you let me know if there are any error messages during the optimization or the final optimization results? It would also be very helpful if you could share the current content of your `optim_reparo.py` file.

**Comment by wolfball** [comment]
> OK, could you let me know if there are any error messages during the optimization or the final optimization results? It would also be very helpful if you could share the current content of your `optimal_reparo.py` file.

Thanks for your kind response!

I just noticed that the loss remained unchanged during the training process. Besides, could you explain the differences between https://github.com/jkxing/pytorch3d and the official version of PyTorch3D? Also, I might be a bit busy recently, so I may reach out to you for a discussion in about a week.

**Comment by VincentHancoder** [comment]
No problem, you can take your time and elaborate on your observations and questions when it's convenient for you.

**Comment by wolfball** [comment]
I ran

```bash
python experiments/furniture/optim_reparo.py --config config/dreamgaussian/furniture_rgbxy_wo_proxy.json
```

and got an image located at "results/dreamgaussian/furniture_rgbxy_wo_proxy/GSO_CHILDREN_BEDROOM_CLASSIC/000/init_img_nv_0.png". However, the image is completely white.

Is this the expected result?
